web/src/pages/MyTransactions/TransactionDetails/PreviewCardButtons/AcceptSettlementButton.tsx

```diff
@@ -39,7 +39,14 @@ const AcceptButton: React.FC = () => {
     }
   };
 
-  return <Button isLoading={isSending} disabled={isSending} text="Accept" onClick={handleAcceptSettlement} />;
+  return (
+    <Button
+      isLoading={isSending}
```

Comment:
We can better load / disable the button by using info from the simulate hook.
While the config is loading, the txn will fail if clicked. or take alot of time.
So it's better to disable when : 
- Config is loading : use `isLoading`
- There is error while simulating : use `isError`